staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -171,6 +171,10 @@ type Creater interface {
 
 	// Create creates a new version of a resource.
 	Create(ctx genericapirequest.Context, obj runtime.Object) (runtime.Object, error)
+
+	// CreateInitialized creates a new version of a resource and waits until the object has been
+	// successfully initialized.
```

Comment:
That's not the contract though.  Maybe my comment misled before - the code in handlers allows for a caller to bypass initialization, in which case it's a 202.